src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/culling_of_stratholme.cpp

```diff
@@ -1251,7 +1256,7 @@ void npc_arthas::npc_arthasAI::ReorderInstance(uint32 data)
                 waveGroupId = 4;
                 SendNextWave(NPC_MEATHOOK);
             }
-            else if (data == COS_PROGRESS_KILLED_SALRAMM)
+            else // if (data == COS_PROGRESS_KILLED_SALRAMM)
```

Comment:
how the logic can be the same?

Reply:
@Helias 
```cpp
case COS_PROGRESS_FINISHED_CITY_INTRO:
case COS_PROGRESS_KILLED_MEATHOOK:
case COS_PROGRESS_KILLED_SALRAMM:
```

because if it's not `COS_PROGRESS_FINISHED_CITY_INTRO` or `COS_PROGRESS_KILLED_MEATHOOK`, then it must be `COS_PROGRESS_KILLED_SALRAMM`